### ISSUE TITLE
Update text for senior housing question on eligiblity questionnaire

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#849]
There is text below the senior housing question on the eligibility questionnaire that currently reads 'Some properties only accept individuals over 62 years of age.' This should be updated to read 'You can choose multiple age ranges. Some rentals have minimum age requirements.'
